plugins/core/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -1517,6 +1517,8 @@ q.onboarding.codewhisperer.description=Amazon Q inline suggestions are also enab
 q.onboarding.description=Amazon Q is your generative-AI powered assistant.
 q.onboarding.title=Meet Amazon Q
 q.reauthenticate=Re-authenticate to connect
+q.session_configuration=Extend your IDE sessions
+q.session_configuration.description=Your maximum session length for Amazon Q can be extended to 90 days by your administrator. For more information, refer to How to extend the session duration for Amazon Q in the IDE in the IAM Identity Center User Guide.
```

Comment:
Do we need the second sentence if we are anyway showing the learn more link?

Reply:
call to action should be to ask the user to provide the info to their admin

Reply:
this messaging comes from product, yes it does say it can be "extended to 90 days by your administrator", is the feedback that the second sentence is redundant?

Reply:
im fine if product approved wording already

it's just awkward because we're telling the user to go there, but it would make no sense to them because they do not have access to the admin console

Reply:
the idea is probably they can pass this info to the admin